nucypher/characters/lawful.py

```diff
@@ -707,9 +708,8 @@ def get_ritual_from_id(self, ritual_id):
 
     def threshold_decrypt(
         self,
-        ritual_id: int,
-        ciphertext: Ciphertext,
-        conditions: Lingo,
+        ritual_id: int,  # TODO should ritual ID be in the DkgMessageKit/ACP? Feels that way
+        dkg_message_kit: DkgMessageKit,
```

Comment:
Looks like ACP is gaining an alias "DKG Message Kit"? Is this somehow different than "Access Control Policy"?

Reply:
> Looks like ACP is gaining an alias "DKG Message Kit"? Is this somehow different than "Access Control Policy"?

Not really.

I'm working off of #3187:
```
MessageKit = FerveoCiphertext + ACP
```

and https://github.com/nucypher/ferveo/issues/147#issuecomment-1645316119 which further refines `FerveoCiphertext` to be:
```
MessageKit = Thin ferveo ciphertext + symmetric ciphertext + ACP
```

We don't currently have a concept of `Thin ferveo ciphertext + symmetric ciphertext`, which @cygnusv is starting to look at, so at the moment I'm just generalizing so that work can be done in parallel. Ultimately, there is a separation between "Message kit" and ACP, but the ACP is contained within the MessageKit.

The goal is to prevent the whole Capsule-MessageKit separation we did for PRE, and not really have the user care about the inner workings of the MessageKit and our use of symmetric encryption underneath. The user passes the message kit, and we use the internal parts of it. 

> (think more ThresholdMessageKit)

Yep. I can rename it.

Reply:
👍🏻 All makes sense - thanks the the explainer.